pyaedt/hfss3dlayout.py

```diff
@@ -144,12 +147,27 @@ def __init__(
             machine,
             port,
             aedt_process_id,
+            ic_mode,
         )
         ScatteringMethods.__init__(self, self)
 
     def _init_from_design(self, *args, **kwargs):
         self.__init__(*args, **kwargs)
 
+    @property
+    def ic_mode(self):
+        """Get/Set the ic mode of current design.
```

Comment:
```suggestion
        """IC mode of current design.
```
Alex K. always told me anything with an @property decorator should be described as a noun. If it was both a getter and setter and this wasn't clear, examples could be provided. How is the return value a Boolean?